### ISSUE TITLE
Massive footprint!

### ISSUE THREAD

**Issue by jordangottlieb-ych** [almasaeed2010/AdminLTE#1901]
EDIT: before adding AdminLTE to laravel project, took _seconds_ to npm run prod, now takes **minutes**.

### Issue type:
- [ ] Feature request 
- [x] Bug report       
- [ ] Documentation   

### Environment:
* AdminLTE Version:  2.4

### Description:
![image](https://user-images.githubusercontent.com/33668261/41443636-fbcb95ce-6ff2-11e8-8e29-2f6a31e9ef43.png)

**Comment by almasaeed2010** [comment]
If you purchase any admin theme, you are likely to find that most of them exceed 70MB. That doesn't mean your final product will have all of those files. Unfortunately, we have to add all of these files to make sure everyone has the required scripts to create what they'd like.

I do admin that we made a mistake with including bower_components but we can't go back on that decision now. I've recently tested v3-alpha in a laravel project and it compiles fast and nicely.

Thanks and sorry for the inconvenience!
